cohere/client.py

```diff
@@ -597,25 +597,29 @@ def _request(self, endpoint, json=None, method="POST", stream=False) -> Any:
     def create_cluster_job(
         self,
         embeddings_url: str,
-        threshold: Optional[float] = None,
         min_cluster_size: Optional[int] = None,
+        n_neighbors: Optional[int] = None,
+        is_deterministic: Optional[bool] = None,
     ) -> CreateClusterJobResponse:
         """Create clustering job.
 
         Args:
             embeddings_url (str): File with embeddings to cluster.
-            threshold (Optional[float], optional): Similarity threshold above which two texts are deemed to belong in
                 the same cluster. Defaults to None.
-            min_cluster_size (Optional[int], optional): Minimum number of elements in a cluster. Defaults to None.
-
+            n_neighbors (Optional[int], optional): Number of neighbors in dimensionality reduction with UMAP. Lower
+                values focus UMAP on local structure of the data while higher values will focus UMAP on global
+                structure of the data. Defaults to None.
```

Comment:
Do we want to tell people what None translates to?

Reply:
why no make the default value the integer values used by default instead of using None?
for n_neighbors we can say "Number of nearest neighbors used by UMAP to establish the local structure of the data" and link to https://umap-learn.readthedocs.io/en/latest/parameters.html#n-neighbors

Reply:
@sanderland is there a usual method to specify defaults values in the API (blobheart code) and access them here via na import (e.g. from cohere.responses.cluster import ClusterDefaults)?  That way we can change in just 1 place?

Reply:
The usual is to have a default of `None`, but mention the API default in the docstring, see e.g. generate.

Reply:
Updated docstring with default values!